app/views/shared/_in-person-verification-results-email-lower.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="mt-40">
+  <table class="button expanded large radius">
+    <tbody>
+      <tr>
+        <td>
+          <table>
+            <tbody>
+              <tr>
+                <td style="text-align: center">
+                  <%= link_to t('user_mailer.in_person_verified.sign_in'),
+                              idv_url,
+                              target: '_blank',
+                              class: 'float-center',
+                              rel: 'noopener' %>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </td>
+        <td class="expander"></td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+<p>
+  <%= link_to idv_url, idv_url, target: '_blank', rel: 'noopener' %>
```

Comment:
Just to note, we're telling the user to follow these links to "try again", but unless the user is already signed in when they visit `/verify`, they would be presented with the sign in page, and we don't currently redirect the user to their original intended destination (LG-3737), so they would arrive at their account page after signing in.